server/src/main/java/org/opensearch/common/blobstore/BlobMetadata.java

```diff
@@ -32,11 +32,14 @@
 
 package org.opensearch.common.blobstore;
 
+import org.opensearch.common.annotation.ExperimentalApi;
+
 /**
  * An interface for providing basic metadata about a blob.
  *
  * @opensearch.internal
  */
+@ExperimentalApi
```

Comment:
We need to make `RemotePinnedTimestampService` as either experiment or public to let ITs go through. Else ITs fail with error `java: The element org.opensearch.node.remotestore.RemoteStorePinnedTimestampService is part of the public APIs but is not marked as @PublicApi, @ExperimentalApi or @DeprecatedApi (referenced by org.opensearch.repositories.Repository)`. Since the dependent classes need to be marked experimental too, had to mark experimental a couple of classes.

Reply:
We have to make `RemotePinnedTimestampService` as `ExperimentalApi` . Other classes won't change as they are released .  We have to find a way to do that . Else we won't be able to backport these changes to 2.x .